jobs/bump-lockfile.Jenkinsfile

```diff
@@ -35,86 +36,140 @@ try { lock(resource: "bump-${params.STREAM}") { timeout(time: 120, unit: 'MINUTE
     """)
 
     def branch = params.STREAM
-    def timestampOnly = false
+    def forceTimestamp = false
+    def haveChanges = false
     shwrap("cosa init --branch ${branch} https://github.com/${repo}")
     shwrap("cosa buildprep ${BUILDS_BASE_HTTP_URL}/${branch}/builds")
 
-    def prevPkgChecksum = shwrapCapture("jq -c .packages src/config/manifest-lock.x86_64.json | sha256sum")
-    def prevPkgTimestamp = shwrapCapture("jq -c .metadata.generated src/config/manifest-lock.x86_64.json | xargs -I{} date --date={} +%s") as Integer
-
+    def lockFile, PkgChecksum, PkgTimestamp
+    def archinfo = [x86_64: [:], aarch64: [:]]
+    for (arch in archinfo.keySet()) {
+        lockFile = "src/config/manifest-lock.${arch}.json"
+        if (utils.path_exists("${lockFile}")) {
+            PkgChecksum = shwrapCapture("jq -c .packages ${lockFile} | sha256sum")
```

Comment:
NIT: `readJSON` can be used here instead of the shell out to `jq`. However, since the shell out is common, its a nit.

Reply:
Since I was adapting what was there previously and we're also doing a checksum I'll leave it be.